src/components/SectioningContent/SectioningContent.stories.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { StoryFn } from '@storybook/react'
+
+import { Heading } from '../Heading'
+
+import { Article, Aside, Nav, Section, SectioningFragment } from './SectioningContent'
+
+export default {
+  title: 'SectioningContent',
```

Comment:
WANT: 何らかのカテゴライズをしたい。

```suggestion
  title: 'Navigation（ナビゲーション）/SectioningContent',
```